README.md

```diff
@@ -146,3 +146,61 @@ You can check out the recorded demo to see how the navigator works.
       <img src="https://i.imgur.com/JtZvOcu.png" alt="Gamepad Navigator Demo" />
    </a>
 </p>
+
+## Publishing to the Chrome Web Store
+
+1. Prepare the code and the `master` branch.
+   1. Increase the `version` number in the [manifest](src/manifest.json#L6) file and push changes to the `master`
+      branch.
+   2. Ensure that all of the code that should be published has been merged into the `master` branch.
+   3. Ensure that the code in the `master` branch is working as expected.
+      1. Run tests: `npm run test`
+      2. Lint: `grunt lint`
+      3. Manual test build.
+         1. Create a build and load the generated unpacked extension into Chrome.  
+            (Refer to the [Installation](#installation) section for more details)
+         2. Test all of the available configuration options (Action, Speed Factor, et cetera) and ensure that they work
+            on the browser correctly.
+            1. Refresh any browser tabs/windows that were open before installing the extension.
+            2. The actions and the associated configuration options should be tested individually and in combinations
+               to ensure that they are working correctly. For example, testing **Scroll Vertically** separately, then
+               changing its **Speed Factor**, and selecting the **Invert Action** option.
+            3. Test the same action and the associated configuration options with at least one different button,
+               thumbstick, and trigger.
+            4. Multiple web pages should be tested to ensure that everything works correctly.
+
+2. Create the release package.
+   1. Create a release build: `grunt build`
+   2. Create a zip archive of the contents of the `dist` directory. The zip file will be uploaded to the
```

Comment:
The creation of the zip should probably be a new Grunt task or an npm script.  That way there's less risk of typos or other human errors in the process.